src/kv/raw_serialise.h

```diff
@@ -210,7 +210,7 @@ namespace kv
       else if constexpr (std::is_same_v<T, kv::EntryType>)
       {
         uint8_t entry_type = read_entry<uint8_t>();
-        if (entry_type > 2)
+        if (entry_type > 4)
```

Comment:
This is an obscure error next time we add an EntryType, would be nice to handle it automatically.

```suggestion
        if (entry_type > kv::EntryType::MAX)
```
and add `MAX = WriteSetWithCommitEvidenceAndClaims` to the enum definition.

Reply:
Done